community/kernel/src/test/java/org/neo4j/kernel/impl/store/CommonAbstractStoreTest.java

```diff
@@ -193,7 +193,8 @@ public void recordCursorPinsEachPageItReads() throws Exception
     {
         File storeFile = dir.file( "a" );
         RecordingPageCacheTracer tracer = new RecordingPageCacheTracer( Pin.class );
-        PageCache pageCache = pageCacheRule.getPageCache( fileSystemRule.get(), tracer, Config.empty() );
+        PageCache pageCache = pageCacheRule.getPageCache( fileSystemRule.get(),
+                config().withTracer( tracer ), Config.empty() );
```

Comment:
It is true that the Tools compile failure is unrelated but you still have some merge issues. `Config.empty()` no longer exists and the PR builder won't tell you that because it is building your branch and not the merge result. So a rebase/merge is still required.